### PR TITLE
Add documentation about policy generator expanders

### PR DESCRIPTION
**Pull request by dhaiducek**
- https://github.com/open-cluster-management/backlog/issues/18203

### CODE REVIEW

**Review by mprahl — DISMISSED** [review]

**Review by ycao56 — APPROVED** [review]

**Review by ycao56 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -71,15 +71,17 @@ again or simply wait for the subscription controller to synchronize the resource
 ### Policy Generator
 
 GitOps through Open Cluster Management is able to handle Kustomize manifests, so you can also use
-the [Policy Generator](https://github.com/stolostron/policy-generator-plugin) Kustomize
-plugin to generate policies from Kustomize manifests in your repository.
+the [Policy Generator](https://github.com/stolostron/policy-generator-plugin) Kustomize plugin to
+generate policies from Kustomize manifests in your repository. The Policy Generator handles
```
Comment (mprahl) [inline README.md:75]: @dhaiducek did you mean `Kubernetes manifests`?

Comment (dhaiducek) [reply README.md:75]: Yes--I'll make an update.

**Pull request merged**